README.md

```diff
@@ -89,6 +89,26 @@ TIME=true HTML=true APP_ENV=test bundle exec rake test
 ```
 This generates a report with test times. You can find the report in optimizer-api/test/html_reports folder.
 
+
+If you want to run a specific test file (let's say real_cases_scheduling_test.rb file only):
+```
+APP_ENV=test bundle exec rake test TEST=test/real_cases_scheduling_test.rb
+```
+If you want to run only one specific test (let's say test_instance_clustered test only) you can use `focus` or call:
+```
+APP_ENV=test bundle exec rake test TESTOPTS="--name=test_instance_clustered"
+```
+Tests are splitted into several sets and you can replay only one or more set(s):
+```
+APP_ENV=test bundle exec rake test:api
+APP_ENV=test bundle exec rake test:api test:models
```

Comment:
I think, after your corrections we don't need the `APP_ENV=test` anymore.

Also, as you suggested, if the correct version of `rake` is the default then `bundle exec` is not necessary either but I guess this way is more secure..

Reply:
yes we don't need, as `bundle exec` in some cases, but I prefer to keep them in README to reveal the different ways to run tasks